tests/general/LineCountBearTest.py

```diff
@@ -25,3 +26,35 @@ def test_run(self):
             filename='default')
         self.check_validity(self.uut, ['1 line'])
         self.check_validity(self.uut, [])  # Empty file
+
+    def test_min_lines_per_file_1(self):
+        self.section.append(Setting('max_lines_per_file', 10))
+        self.section.append(Setting('min_lines_per_file', 5))
+        self.check_results(
+            self.uut, ['line 1', 'line 2', 'line 3', 'line 4'],
+            [Result.from_values('LineCountBear',
+                                'This file had 4 lines, which is 1 lines less '
+                                'than the minimum limit specified.',
+                                severity=RESULT_SEVERITY.NORMAL,
+                                file='default')],
+            filename='default')
+
+    def test_min_lines_per_file_2(self):
+        self.section.append(Setting('max_lines_per_file', 10))
+        self.section.append(Setting('min_lines_per_file', 1))
+        self.check_results(
+            self.uut, [],
+            [Result.from_values('LineCountBear',
+                                'This file had 0 lines, which is 1 lines less '
+                                'than the minimum limit specified.',
+                                severity=RESULT_SEVERITY.NORMAL,
+                                file='default')],
+            filename='default')
+
+    def test_min_lines_per_file_3(self):
+        self.section.append(Setting('max_lines_per_file', 10))
+        self.section.append(Setting('min_lines_per_file', 1))
+        self.check_results(
+            self.uut, ['line 1', 'line 2', 'line 3', 'line 4', 'line 5'],
+            [],
+            filename='default')
```

Comment:
It would be good if you could add a test in which `Setting('max_lines_per_file', 5)` , `Setting('max_lines_per_file', 5)` and `file_length = 5` so both cases fail and both messages are printed. :)